docs/src/atopics.md

```diff
@@ -0,0 +1,335 @@
+# Advanced Topics
+
+## Live plotting
+
+Plotting while a computational heavy simulation is performed can be easily achieved with FerriteVis.jl.
+Every plotter object of type `MakiePlotter` holds a property called `u` which is a so called `Observable`.
+If an `Observable` changes, all its dependencies are triggered to change as well. So, all we need to do is to update
+the observable `plotter.u`.
```

Comment:
Does this belong here? I mean it is a technical detail and should more belong to the plotter docs, right?

Reply:
it's just for now until the plotter is somewhat stable :)